src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/ExportAzureRmDataLakeStoreItem.cs

```diff
@@ -95,20 +116,59 @@ public override void ExecuteCmdlet()
                         throw new CloudException(string.Format(Resources.InvalidExportPathType, Path.TransformedPath));
                     }
 
-                    if (type == FileType.FILE)
+                    DataLakeStoreTraceLogger logger = null;
```

Comment:
Suggestion: For the future, you'll need to think about how this would need to be augmented to support muyltiple cmdlet executions simultaneously in an AppDomain.  In this case, you would want to differentiate which traces go to which log file.